src/config/dynamic-theme-fixes.config

```diff
@@ -17100,6 +17100,64 @@ CSS
 
 ================================
 
+upwork.com
+/*---------------Home page fixes-------------------*/
```

Comment:
Please remove comment, they aren't allowed to be in the fixes file

Reply:
is that correct

Reply:
Yes.